services/lsst-dev.rst

```diff
@@ -89,105 +92,81 @@ For more information on using SSH public/private keys:
+You may wish to automatically enable ``devtoolset-3`` every time you log in to ``lsst-dev`` by adding it to your shell initialization files.
+For example, try adding the following to :file:`~/.profile`:
 
-.. _lsst-dev-alt-tools:
+.. code-block:: bash
 
-Alternate Development Tools
----------------------------
+   exec scl enable devtoolset-3 bash
 
-Developer Toolset is an offering for developers on EL (Red Hat, CentOS, SCL) distributions.
-Using a framework called Software Collections, an additional set of tools is installed into the /opt directory, as recommended by the UNIX Filesystem Hierarchy Standard.
-These tools are enabled by the user on demand using the supplied scl utility.
+.. _GNU Compiler Collection: https://gcc.gnu.org/
+.. _prerequisites for building the LSST stack: https://confluence.lsstcorp.org/display/LSWUG/OSes+and+Prerequisites
+.. _Red Hat Developer Toolset: http://developers.redhat.com/products/developertoolset/overview/
+.. _Git: https://www.git-scm.com/
 
-Developer Toolset 3.x provides following tools:
+.. _lsst-dev-loadlsst:
 
-- gcc/g++/gfortran - GNU Compiler Collection - version 4.9.2
-- git - GIT - version 1.9.3
-- gdb - GNU Debugger - version 7.8.2
-- binutils - A GNU collection of binary utilities - version 2.24
-- elfutils - A collection of utilities and DSOs to handle compiled objects - version 0.161
-- dwz - DWARF optimization and duplicate removal tool - version 0.11
-- systemtap - Programmable system-wide instrumentation system - version 2.6
-- valgrind - Tool for finding memory management bugs in programs - version 3.10.1
-- oprofile - System wide profiler - version 0.9.9
+Load the LSST Environment
+=========================
 
-To test the alternate devtoolset environment:
+A ‘shared’ installation of the LSST software stack is installed under :file:`/ssd/lsstsw/stack`.
+This installation is regularly updated to recently weekly builds of the ``lsst_distrib`` top-level package; the most recent build is tagged as ``current``.
+Add this shared stack to your environment and set up the latest build of the LSST applications by running:
 
-.. code-block:: bash
+.. prompt:: bash
 
-   $ scl enable devtoolset-3 bash
-   $ gcc --version
-   gcc (GCC) 4.9.2 20150212 (Red Hat 4.9.2-6)
-   Copyright (C) 2014 Free Software Foundation, Inc.
-   This is free software; see the source for copying conditions.  There is NO
-   warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  source /ssd/lsstsw/stack/loadLSST.bash
+  setup lsst_apps
 
-To make use of Git 1.9.x you can also use the Git 1.9 software collection.
-You can enable multiple software collections at the same time, so the following will enable both the ``devtoolset-3`` and ``git19``: 
+(substitute :file:`loadLSST.csh`, :file:`loadLSST.ksh` or :file:`loadLSST.zsh`, depending on your preferred shell).
 
-.. code-block:: bash
+Since this stack is shared, all members of the ``lsst`` group have permission to declare new products within it.
```

Comment:
Could you add a sentence or two explaining why one would want to declare a product in the shared stack? I don't actually know myself; is this if I've built a new version in my home directory and want to share it?

Is "[When sharing a EUPS_DIR with others, what’s the polite way to declare packages?](https://community.lsst.org/t/when-sharing-a-eups-dir-with-others-whats-the-polite-way-to-declare-packages/97)" relevant here?